site/_docs/howto.md

```diff
@@ -684,8 +684,14 @@ Note: release artifacts (dist.apache.org and repository.apache.org) are managed
 
 Before you start:
 
+* Send an email to [dev@calcite.apache.org](mailto:dev@calcite.apache.org) notifying that RC build process
+  is starting and therefore `master` branch is in code freeze until further notice.
 * Set up signing keys as described above.
+* Upload your key to [https://keyserver.ubuntu.com](https://keyserver.ubuntu.com) and/or
+  [http://pool.sks-keyservers.net:11371](http://pool.sks-keyservers.net:11371) (keyservers used by Nexus).
 * Make sure you are using JDK 8 (not 9 or 10).
+* Make sure `master` branch and `site` branch are in sync, i.e. there is no commit on `site` that has not
+  been applied also to `master`. If this the case, cherry-pick the missing commit(s) into `master`.
```

Comment:
You can check by doing something like 'git checkout site; git rebase -i master'.